src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -1213,6 +1214,38 @@ class spell_pal_seal_of_righteousness : public SpellScriptLoader
         }
 };
 
+// Blinding Light - 115750
+class spell_pal_blinding_light : public SpellScriptLoader
+{
+    public:
+        spell_pal_blinding_light() : SpellScriptLoader("spell_pal_blinding_light") { }
+
+        class spell_pal_blinding_light_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_pal_blinding_light_SpellScript);
+
+            void HandleOnHit()
+            {
+                if (Player* _player = GetCaster()->ToPlayer())
+                {
```

Comment:
Not a problem. Just a question from Keader if it is according to coding standards. :-)